core/eolearn/tests/test_eodata_io.py

```diff
@@ -660,6 +660,30 @@ def test_partial_temporal_saving_into_existing(eopatch: EOPatch, temporal_select
         assert_array_equal(loaded_patch.timestamps, eopatch.timestamps)
 
 
+@mock_s3
+@pytest.mark.parametrize("dtype", [float, np.float32, np.int8])
```

Comment:
Would it make sense to parametrize few cases of `temporal_selection` (or e.g. the number of timestamps to be saved?)

Reply:
The `test_partial_temporal_saving_into_existing` covers multiple cases, here I just wanted to test that 'infer' really does initialize an array if it doesn't exist